README.md

```diff
@@ -2,7 +2,7 @@
 
 __`on-http` is the HTTP server for RackHD__
 
-_Copyright 2015, EMC, Inc._
+_Copyright 2016, EMC, Inc._
```

Comment:
I don't think the copyright should be changed. I remembered the Copyright 2015 means the copyright 2015 to now.